### ISSUE TITLE
Indentation bug on array

### ISSUE THREAD

**Issue by SaschaNaz** [Microsoft/TypeScript#9437]
**TypeScript Version:**  master branch

**Code**

Copy-pasting #7503 but with a slight change...

```ts
export let Things = [3, {
    Hat: 'hat',
    Glove: 'glove',
    Umbrella: 'umbrella'
},
    {
        Salad: 'salad',
        Burrito: 'burrito',
        Pie: 'pie'
    }];
```

**Expected behavior:**

```ts
export let Things = [3, {
    Hat: 'hat',
    Glove: 'glove',
    Umbrella: 'umbrella'
},
{
    Salad: 'salad',
    Burrito: 'burrito',
    Pie: 'pie'
}];
```

**Actual behavior:** No change

**Comment by SaschaNaz** [comment]
Should be fixed by #3827.

**Comment by SaschaNaz** [comment]
Duplicate of #8644 which also is fixed by #3827.
